packages/openapi3/src/lib.ts

```diff
@@ -30,7 +30,13 @@ export const libDef = {
     "duplicate-response": {
       severity: "error",
       messages: {
-        default: paramMessage`Multiple return types for status code ${"statusCode"}`,
+        default: paramMessage`Multiple return types for content type ${"contentType"} and status code ${"statusCode"}`,
```

Comment:
Not for this PR but what we were talking about: Should that validation technically be moved to the rest layer?